doc/data/messages/f/fixme/good.py

```diff
@@ -1 +1 @@
-# This is a placeholder for correct code for this message.
+# TODO: Some kind of an activity
```

Comment:
```suggestion
```

Reply:
```suggestion
# I really need to fix this bug!
```

Reply:
The check does not make a lot of sense though. A TODO is better than a raw comment. This is what we were doing until we deactivated the check:
```suggestion
# pylint: disable-next=fixme
# TODO: We really need to fix this bug!
```